### PR TITLE
Add 'aws-sdk' installation instructions

### PR DESCRIPTION
**Pull request by topher200**
Without installing the sdk first I get this error:

```
$ node index.js
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'aws-sdk'
```

After installing the sdk everything works fine!

### CODE REVIEW

**Comment by Alynva** [comment]
In most NodeJs projects, you have to install de dependencies of it with `npm install` before running the project. The `aws-sdk` is already listed as dependency on the `package.json`, so no need to specify witch dependency you have to install, just do the `npm install` and you're good to go. But I agree that is better to add this in the README, just don't need the `aws-sdk` in it.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # aws-cloudwatch-metric-count
 
-To run it just type:
+To run it, simply clone this repo and then:
 
+    npm install aws-sdk
```
Comment (gvasquez95) [inline README.md:5]: the `aws-sdk` part is redundant, `npm instal`l is enough

**Review by gvasquez95 — CHANGES_REQUESTED** [review]

**Comment by millerick** [comment]
I rolled these changes into another update to the README I thought of
https://github.com/jooycar/aws-cloudwatch-metric-count/pull/10

**Comment by millerick** [comment]
I rolled these changes into another update to the README I thought of
https://github.com/jooycar/aws-cloudwatch-metric-count/pull/10

**Pull request closed without merging**